### DISCUSSION TITLE
Welcome to ghidra Discussions!

### DISCUSSION THREAD

**Discussion by emteere** [NationalSecurityAgency/ghidra#2563] [General]
![GHIDRA_1](https://user-images.githubusercontent.com/47253321/102395791-9b275080-3fa9-11eb-878e-a12820ddf22a.png)
## Welcome!

With the new year fast approaching and a preview feature on the way, now seemed like a good time to open github Discussions.

We’re hoping to use Discussions as a place to better manage our connection with the community.  We hope you will:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and be open-minded.
  
Please bear with us as the team gets used to this new discussion area.  The organization will likely evolve over time.  If there are bugs or issues you are encountering please post them in the Issues section.  For questions and general discussions about ghidra features please open a discussion here.  We'll monitor and respond as best we can.

**Comment by MaheraMihanta** [comment]
bonjour tout le monde, je suis sur une décompilation avec ghidra, je ne connais pas vraiment l'assembleur, par contre je peux tout à fait me débrouiller en python, ou autre language, ma question: comment transcrire le résultat de la décompilation issu de ghidra en Python? à la limite en C ou C++ ou Java

**Comment by d-millar** [comment]
Les résultats de la décompilation devraient être proches du C compilable. Rendre les résultats dans quelque chose de plus proche de Python serait tout un projet et n'est pas actuellement sur notre radar. (Toutes mes excuses par avance pour l'utilisation de Google Translate.)

**Comment by MaheraMihanta** [comment]
Thank you for paying attention to the comment I left. Currently, I am
working on decompiling some software from the company where I work. These
are really old software, written in 16 bits. The goal is to recreate the
scenarios, rebuild the graphical interfaces, and all the functionalities
with them. I specialize in Python; however, I am also quite knowledgeable
in C and C++. Once we have succeeded in translating the resulting C code
from Ghidra, we will share the results of the research with you if you want

**Comment by stephen-hocking** [comment]
I'm trying to convert the asm portions of https://github.com/dkfans/keeperfx/blob/master/src/bflib_render_gpoly.c back into C. Have succeeded to a certain point, but I've been hand converting the _DAT_00blah variable references in the decompiled code back into the actual variable names by hand, which is tedious & error prone. Is there an automated way to do this? The .o file is compiled with gcc's -g option, which should put all the debug symbols in.

**Comment by dtardogno** [comment]
Greets to all. I am reversing a group of windows exe's, several of which import the same dll.
I want to organize my projects per exe. It would be really great if the db for a common dll, were accessible/shared across all the exe projects that import it. However, I have seen that this is not the case. Each exe project creates its own separate db for that common dll. Is there any way to achieve this by ghidra path setup or otherwise? Thanks for any insights!
